app/views/shared/_filter_search.html.erb

```diff
@@ -4,22 +4,32 @@
 
   function filterLocations(){
     let filterValue = document.getElementById('filterInput').value.toUpperCase();
-    let table = document.getElementById('wrapper')
-    let location_name = table.querySelectorAll('table.govuk-table');
-    let ips = table.querySelectorAll('tbody#ips-table');
+    let location_name = document.getElementById('wrapper').querySelectorAll('table.govuk-table');
 
     for(let i = 0;i < location_name.length;i++){
       let text = location_name[i].getElementsByTagName('div')[0];
+      location_name[i].style.display = 'none';
 
       if(text.innerHTML.toUpperCase().indexOf(filterValue) > -1){
         location_name[i].style.display = '';
-        ips[i].style.display = '';
-      } else {
-        location_name[i].style.display = 'none';
-        ips[i].style.display = 'none';
       }
     }
+
+    displayResultsMessage(location_name);
+  }
+
+function displayResultsMessage(location_name) {
+  var noResultsMessage = document.getElementById('noLocationResults');
```

Comment:
Should this be changed to `let` instead of `var`?
Feel like I've read that it's better to use `let` or `const` rather than `var`

Reply:
These are very new.  Not sure all browsers will support this (given these are mainly government orgs)